.env

```diff
@@ -10,3 +10,9 @@ HARVEST_SOURCE_URL=http://localhost:${HARVEST_SOURCE_PORT}
 
 CF_API_URL=https://api.fr.cloud.gov
 LM_RUNNER_APP_GUID=f4ab7f86-bee0-44fd-8806-1dca7f8e215a
+
+CLIENT_ID=urn:gov:gsa:openidconnect.profiles:sp:sso:gsa:datagov-dev-harvest-admin
+ISSUER=https://idp.int.identitysandbox.gov
+REDIRECT_URI=http://localhost:8080/callback
+FLASK_APP_SECRET_KEY=Px4o0MPEsCzHJRRSUDyt1VRi-lYF2W5VEzR13lylAgI
```

Comment:
will need to recreate these keys in dev I'm guessing

Reply:
right, for now we just keep it same in local and dev. We will create new one on staging and prod later.